app/renderer/src/assets/locales/en/translation.json

```diff
@@ -281,5 +281,6 @@
   "noResultsFound": "No results found",
   "invalidCapType": "Invalid capability type: {{type}}",
   "whitespaceDetected": "Text Starts and/or Ends With Whitespace",
-  "pcloudyCredentialsRequired": "pCloudy username and api key are required!"
+  "pcloudyCredentialsRequired": "pCloudy username and api key are required!",
+  "duplicateCapabilityNameError": "Capability Set with the same name already present"
```

Comment:
Maybe something like 'A capability set with the same name already exists'

Reply:
@eglitise Sounds good.